packages/design-system/src/components/OcSelect/OcSelect.vue

```diff
@@ -296,7 +301,55 @@ export default defineComponent({
       setComboBoxAriaLabel()
     })
 
-    return { select, userInput }
+    const dropdownEnabled = ref(false)
+    const setDropdownEnabled = (enabled) => {
```

Comment:
Is this helper function really needed ?

Reply:
No, obviously not - but it makes it easier to debug issues because you can set breakpoints or log here.